04 Abstraction Exercises Part 1/delos-reyes-abstraction-exercise/src/main/java/problem/delosreyes/Rectangle.java

```diff
@@ -0,0 +1,34 @@
+package main.java.problem.delosreyes;
+
+public class Rectangle {
+    private float length;
+    private float width;
+    private String unitOfMeasurement;
```

Comment:
Consider what should be done here since this will be the same to all the instances of the Rectangle

Reply:
Talking about unitOfMeasurement